comtypes/test/test_dispifc_safearrays.py

```diff
@@ -0,0 +1,88 @@
+# coding: utf-8
+
+import unittest
+import comtypes
+
+from comtypes import CLSCTX_LOCAL_SERVER
+from comtypes.client import CreateObject, GetModule
+from ctypes import byref, pointer, c_double
```

Comment:
Even though there are no problems at runtime because it has been already loaded, IDEs can warn when referencing symbols that are not imported.
Therefore, I will explicitly import `comtypes.safearray` into this module.

Also, I have arranged the imports in the order recommended by PEP8.

```suggestion
from ctypes import byref, c_double, pointer
import unittest

import comtypes
import comtypes.safearray
from comtypes import CLSCTX_LOCAL_SERVER
from comtypes.client import CreateObject, GetModule
```